File: src/main/java/nl/dannyjelsma/soundboardbot/commands/music/PauseCommand.java
package nl.dannyjelsma.soundboardbot.commands.music;

import com.jagrosh.jdautilities.command.Command;
import com.jagrosh.jdautilities.command.CommandEvent;
import nl.dannyjelsma.soundboardbot.audio.SoundManager;

public class PauseCommand extends Command {

    public PauseCommand() {
        name = "pause";
    }

    @Override
    protected void execute(CommandEvent event) {

        SoundManager manager = SoundManager.getInstance();
        boolean paused = manager.getAudioPlayer().isPaused();

        manager.getAudioPlayer().setPaused(!paused);
        event.getChannel().sendMessage("Set paused to **" + !paused + "**.").queue();
    }
}

File: src/main/java/nl/dannyjelsma/soundboardbot/commands/music/PauseCommand.kt
package nl.dannyjelsma.soundboardbot.commands.music

import com.jagrosh.jdautilities.command.Command
import com.jagrosh.jdautilities.command.CommandEvent
import nl.dannyjelsma.soundboardbot.audio.SoundManager

class PauseCommand : Command() {
    init {
        name = "pause"
    }

    override fun execute(event: CommandEvent) {
        SoundManager.audioPlayer.isPaused = !SoundManager.audioPlayer.isPaused
        event.channel.sendMessage("Set paused to **" + !SoundManager.audioPlayer.isPaused + "**.").queue()
    }
}